wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Countries card

Fixes #11082 

This add the Period Country card. 
- Displays a temporary view for the map area. The map will be added with https://github.com/wordpress-mobile/WordPress-iOS/issues/11083.
- Displays top 6 country rows.
- Rows have no select action.

Note: There is still an outstanding issue with images not loading sometimes on rotation (this is most obvious on iPad). I'll address that in a future PR.

To test:

---
- On a site with views, go to Stats > Period > Countries.
- Verify the view is:
<img width="350" alt="countries" src="https://user-images.githubusercontent.com/1816888/53043077-e96a7780-3444-11e9-9042-85a4238c4ff7.png">

---
- On a site with no views, go to Stats > Period > Countries.
- Verify the view is:
<img width="350" alt="countries_nil" src="https://user-images.githubusercontent.com/1816888/53043233-436b3d00-3445-11e9-9880-31e731a0ffcb.png">
